pallets/xcmp-queue/src/lib.rs

```diff
@@ -472,6 +357,22 @@ impl Default for QueueConfigData {
 	}
 }
 
+impl QueueConfigData {
+	/// Validate all assumptions about `Self`.
+	///
+	/// Should be called prior to accepting this as new config.
+	pub fn validate<T: crate::Config>(&self) -> sp_runtime::DispatchResult {
+		if self.resume_threshold < self.suspend_threshold &&
+			self.suspend_threshold <= self.drop_threshold &&
+			self.resume_threshold > 0
```

Comment:
Should `resume_threshold` also be lesser than `suspend_threshold` (i.e. `1..self.suspend_threshold.contains(self.resume_threshold)`)?

Reply:
WDYM? I check it right here `self.resume_threshold < self.suspend_threshold`.  
And then further down we have the resume condition like `suspended && count <= resume_threshold as u64`.

Reply:
:facepalm: Of course, you're right - sorry for the noise